test/images/sample-apiserver/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
it might be more intuitive to match the k8s api server version (i.e. use 1.9 here).

Reply:
I love the suggestion, will change it - I think I was stuck with the 1.7 being 1.0, and couldnt think about this, thank you!